cdi/embedded/src/test/java/org/infinispan/cdi/embedded/test/distexec/DistributedExecutorCDITest.java

```diff
@@ -62,9 +62,6 @@ public void testInvocationUsingImpliedInputCacheRunnable() throws Exception {
 
    static class SimpleCallable implements Callable<Integer>, Serializable, ExternalPojo {
 
-      /** The serialVersionUID */
-      private static final long serialVersionUID = -8589149500259272402L;
```

Comment:
AFAIK the compiler doesn't generate it automatically, it's computed on the fly:
stackoverflow.com/questions/29081134/why-compiler-do-not-set-serialversionuid-automatically

But the real reason to keep it is that IntelliJ will show an annoying warning if the class is `Serializable` and doesn't have a `serialVersionUID` :)